src/shims/dir.props

```diff
@@ -17,13 +17,16 @@
     <NetFxReference Include="System.ComponentModel.Composition" />
     <NetFxReference Include="System.ComponentModel.DataAnnotations">
       <StrongNameSig>MSSharedLib</StrongNameSig>
-    </NetFxReference> 
+    </NetFxReference>
     <NetFxReference Include="System.Data" />
     <NetFxReference Include="System.Drawing" />
     <NetFxReference Include="System.IO.Compression.FileSystem" />
     <NetFxReference Include="System.Net" />
     <NetFxReference Include="System.Numerics" />
     <NetFxReference Include="System.Runtime.Serialization" />
+    <NetFxReference Include="System.ServiceModel.Web">
```

Comment:
Does everything needed come from this repo?  I would have expected stuff to come from WCF given the name.